crates/mdbook-goals/src/mdbook_preprocessor.rs

```diff
@@ -246,25 +261,23 @@ impl<'c> GoalPreprocessorWithContext<'c> {
         Ok(())
     }
 
+    /// Find the goal documents for the milestone in which this `chapter_path` resides.
+    /// e.g., if invoked with `2024h2/xxx.md`, will find all goal documents in `2024h2`.
     fn goal_documents(&mut self, chapter_path: &Path) -> anyhow::Result<Arc<Vec<GoalDocument>>> {
         // let chapter_path = self.ctx.config.book.src.join(chapter_path);
```

Comment:
pre-existing but 

```suggestion
        // let chapter_path = self.ctx.config.book.src.join(chapter_path);
```

(we can fix it later)